.github/workflows/build_and_test.yml

```
@@ -0,0 +1,64 @@
+name: test
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  ubuntu-build-test:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: src/fastdds_python
+
+      - name: Install apt dependencies
+        run: |
+          sudo apt update
+          sudo apt install -y \
+            wget \
+            python3 \
+            python3-pip \
+            libasio-dev \
+            libtinyxml2-dev \
+            swig \
+            libpython3-dev
+
+      - name: Install python dependencies
+        run: |
+          sudo pip3 install -U \
+            vcstool \
+            colcon-common-extensions
+
+      - name: Fetch repositories
+        run: |
+          wget https://raw.githubusercontent.com/eProsima/Fast-DDS-python/master/fastdds_python.repos
+          vcs import --skip-existing src < fastdds_python.repos
```

Shouldn't we use the repos file on `src/fastdds_python/fastdds_python.repos` ?
```suggestion
          vcs import --skip-existing src < src/fastdds_python/fastdds_python.repos
```